snuba/cli/optimize.py

```diff
@@ -96,8 +96,12 @@ def optimize(
     last_midnight = (datetime.now() + timedelta(minutes=10)).replace(
         hour=0, minute=0, second=0, microsecond=0
     )
+
+    # add 1 hour to make the redis TTL past the optimize job cuttoff time
     cutoff_time = last_midnight + settings.OPTIMIZE_JOB_CUTOFF_TIME
-    logger.info("Cutoff time: %s", str(cutoff_time))
+    redis_expire_time = cutoff_time + timedelta(hours=1)
+    logger.info("Cutoff time: %s" % str(cutoff_time))
```

Comment:
Nit: I usually prefer f-string syntax over this. Not something which blocks merging. Upto you if you want to handle it or not.